tests/python/relax/test_op_transform.py

```diff
@@ -0,0 +1,1421 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+import pytest
+import tvm
+import tvm.testing
+from tvm import relax, tir
+from tvm import TVMError
+from tvm.ir import Op
+from tvm.script import relax as R
+
+
+def test_op_correctness():
+    x = relax.Var("x", R.Tensor((3, 4, 5), "float32"))
+    fill_value = relax.Var("fill_value", R.Tensor((), "float32"))
+    assert relax.op.reshape(x, (4, 5, 3)).op == Op.get("relax.reshape")
+    assert relax.op.transpose(x).op == Op.get("relax.transpose")
+    assert relax.op.expand_dims(x, axis=[]).op == Op.get("relax.expand_dims")
+    assert relax.op.squeeze(x).op == Op.get("relax.squeeze")
+    assert relax.op.flatten(x).op == Op.get("relax.flatten")
+    assert relax.op.cast(x, "float32").op == Op.get("relax.cast")
+    assert relax.op.full(fill_value, (2, 3)).op == Op.get("relax.full")
+    assert relax.op.full_like(x, fill_value).op == Op.get("relax.full_like")
```

Comment:
The order of parameters in `full` is strange. (Although it is the same with Relay) Maybe we can change it to
```
full(shape, fill_value)
```
PS: in torch
```
torch.full(size, fill_value)
torch.full_like(input, fill_value)
```

Reply:
This is because previously full only takes shape from the Attrs. Now that shape is first class, we can instead take shape as the first argument. @MasterJH5574